### ISSUE TITLE
Default keyboard

### ISSUE THREAD

**Issue by kilichenko** [rationalmatter/Juno-Issue-Tracker#74]
It is really annoying that it is not possible to use the default system keyboard with Juno. It is handy to use a built in text replacement which does not work right now.

**Comment by navoshta** [comment]
Hi @kilichenkol, could you please explain what exactly is not working here? Juno should work fine with system keyboard. If you mean autofill suggestions, then those are disabled as they are rarely useful when writing code/markdown.

**Comment by kilichenko** [comment]
You are right it is a system keyboard, just with disabled autocomplete. Is it possible to add a switch in settings to enable it?

**Comment by navoshta** [comment]
@kilichenko I'll see if there's an easy way to restrict this to Markdown cells, or something like that. The main issue I'm anticipating is that it may not play well with cell action buttons that are currently displayed instead of autocomplete suggestions.

**Comment by kilichenko** [comment]
As for me it would also absolutely fine even if suggestions were not displayed but the autocomplete simply worked)
